service/src/PlatformTopo.cpp

```diff
@@ -463,28 +464,56 @@ namespace geopm
     void PlatformTopoImp::create_cache(void)
     {
         try {
-            PlatformTopoImp::create_cache(M_SERVICE_CACHE_FILE_NAME);
+            // TODO: Handle return string?
+            (void) PlatformTopoImp::create_cache(M_SERVICE_CACHE_FILE_NAME);
```

Comment:
I wasn't sure if I should plumb this string all the way through to the CFFI wrappers, so for the moment I'm ignoring it.

If this is all OK, I could do all that plumbing which would make it so we don't have to open/read any files around this in the Python code.  The CFFI wrapper would just return the file's contents.